src/iologindata.cpp

```diff
@@ -44,13 +44,6 @@ Account IOLoginData::loadAccount(uint32_t accno)
 	return account;
 }
 
-bool IOLoginData::saveAccount(const Account& acc)
```

Comment:
why are you removing it?

Reply:
Probably, it's not used in the code, both this and the other function he removed.